signup/tool/src/java/org/sakaiproject/signup/tool/jsf/SignupUIBaseBean.java

```diff
@@ -738,11 +754,13 @@ public String getAllAttendeesEmailAddressesFormatted() {
 		Set<String> emails = new HashSet<String>();
 		
 		StringBuilder sb = new StringBuilder();
-		for (TimeslotWrapper tsWrapper : timeslotWrappers) {
-			for(AttendeeWrapper atWrapper : tsWrapper.getAttendeeWrappers()) {
-				String email = atWrapper.getAttendeeEmail();
-				if(StringUtils.isNotBlank(email)){
-					emails.add(email);
+		if (timeslotWrappers!=null){

```

Comment:
Additionally, there is also the practical issue of bang for buck.  This problem is pretty niche.  It requires someone's session to expire whilst they are viewing an event's details or to pass a bookmarked page to someone else - not really super-crucial issues for the amount of time I've spent on it.